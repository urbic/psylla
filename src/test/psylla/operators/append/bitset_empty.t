bitset dup 11 append ?
