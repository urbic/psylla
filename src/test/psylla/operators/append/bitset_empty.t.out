%bitset=000000000001%
